lib/vanagon/component/dsl.rb

```diff
@@ -184,7 +184,8 @@ def install_service(service_file, default_file = nil, service_name = @component.
           # Return here because there is no file to install, just a string read in
           return
         when "windows"
-          # Placeholder for Windows
+          @component.service = OpenStruct.new(:name => service_name, :service_file => service_file)
```

Comment:
like, what's the user going to put into their component file when then need to add a service? What does this look like when it's being used?